Formula/spidermonkey.rb

```diff
@@ -20,49 +21,52 @@ class Spidermonkey < Formula
     sha256 cellar: :any, mojave:   "8c0b46bc04a7e95f99262969b22cc311ee1f7d83413af05865318743ccd96944"
   end
 
-  depends_on :macos # Due to Python 2
+  depends_on "autoconf@2.13" => :build
+  depends_on "pkg-config"    => :build
+  depends_on "python@3.9"    => :build
+  depends_on "rust"          => :build
+
+  depends_on "icu4c"
   depends_on "nspr"
-  depends_on "readline"
 
-  conflicts_with "narwhal", because: "both install a js binary"
+  uses_from_macos "libedit"
```

Comment:
Why not use `readline`?

Reply:
There is a libreadline in [sdk](https://github.com/phracker/MacOSX-SDKs/blob/master/MacOSX11.3.sdk/usr/lib/libreadline.tbd)

Reply:
I'm aware of that, but I'm not sure that explains why we should change it...